pages/workspaceLogs/workspaceLogs.sql

```diff
@@ -0,0 +1,23 @@
+-- BLOCK select_workspace_logs
+SELECT
+    date,
+    format_date_full_compact(date, $display_timezone) AS date_formatted,
+    message,
+    version,
+    state
+FROM workspace_logs
+WHERE workspace_id = $workspace_id
+ORDER BY date ASC, id ASC;
+
+-- BLOCK select_workspace_version_logs
+SELECT
+    date,
+    format_date_full_compact(date, $display_timezone) AS date_formatted,
+    message,
+    version,
+    state
+FROM workspace_logs
+WHERE
+    workspace_id = $workspace_id
+    AND version = $version
```

Comment:
If you want to combine these the two queries in this file you can do `AND ($version IS NULL OR version = $version)` and then pass `version: null` for the case where you want all the logs.

Reply:
Done in 3ec08ea!